core/src/test/java/org/hibernate/ogm/backendtck/type/Bookmark.java

```diff
@@ -63,9 +63,14 @@
 	@Type(type = "numeric_boolean")
 	private boolean isShared;
 
-	// byte arrays
+	// Lobs
 	@Lob
 	private byte[] lob;
+
+	@Lob
+	private String lobWithString;
```

Comment:
Could you add a test for other basic types (e.g. long) and an element collection thereof, too?